pkg/deploy/lattice/service_network_synthesizer.go

```diff
@@ -44,34 +49,33 @@ func (s *serviceNetworkSynthesizer) Synthesize(ctx context.Context) error {
 	if ret != "" {
 		return errors.New(ret)
```

Comment:
do you mind to cleanup this error handling too?

Reply:
I gave this a try, but after a couple of hours of dealing with e2e test failures, reverted it for the sake of timeboxing. It seems as though we have some reliance on this error handling logic, which I'd like to keep outside the scope of the logging changes